### PR TITLE
Experience/13664/roadmap

### LINKED ISSUE #13664: Engineering work to add roadmap to website
## Problem statement
Add the product roadmap to the RS website

## What you need to know
- The content is in this [Roadmap doc](https://cdc.sharepoint.com/:w:/r/teams/ReportStream/_layouts/15/Doc.aspx?sourcedoc=%7BC9FF05CA-C854-458D-8A78-2A5BD2CA50E6%7D&file=20240301_Public_Roadmap.docx&action=default&mobileredirect=true)
- **Do not use** Figma for content source of truth

## Acceptance criteria
- [x] New page created in accordance with [design artifacts](https://www.figma.com/file/86JATe4hOwPfihwwloM5b8/Blockframes?type=design&node-id=2131-1946&mode=design)
- [x] Content matches [Roadmap doc](https://cdc.sharepoint.com/:w:/r/teams/ReportStream/_layouts/15/Doc.aspx?sourcedoc=%7BC9FF05CA-C854-458D-8A78-2A5BD2CA50E6%7D&file=20240301_Public_Roadmap.docx&action=default&mobileredirect=true)
- [x] Page appears in the side nav on About page
- [x] Page appears in top nav About drop-down
- [x] E2E tests are created
  - [x] Page loads
  - [x] Page content loads without errors
  - [x] All Internal Links work and take you to expected page
  - [x] All External Links work
  - [x] Components work as expected (eg, accordion expands/collapses, side nav, etc)

### PR DESCRIPTION
**Pull request by penny-lischer**
This PR ...

Add the product roadmap to the RS website

Test Steps:
1. Log into RS
2. Click on About
3. Click on roadmap

## Changes
Screenshot 2024-03-18 at 10 47 27 AM
Screenshot 2024-03-18 at 10 47 31 AM
Screenshot 2024-03-18 at 10 47 40 AM

### Testing
- [x] Tested locally?
- [ ] Ran `./prime test` or `./gradlew testSmoke` against local Docker ReportStream container?
- [x] (For Changes to /frontend-react/...) Ran `npm run lint:write`? 
- [ ] Added tests?

## Linked Issues
- Fixes #13664

### CODE REVIEW

**Review by jpandersen87 — APPROVED** [review]

**Comment by jillian-hammer** [comment]
thank you @penny-lischer this looks great! here are my visual design notes cc @audreykwr 

- [ ] change color of line rules to `base-lighter` 
- [ ] apply padding to the line rules (padding-top: 32px padding-bottom: 32px)

**Comment by audreykwr** [comment]
- [ ] Let's go with the base-lighter for the lines under the H2s as noted in the handoff doc
- [ ] Make the nav locations say `Product roadmap` instead of just `Roadmap` (that might have only been reflected in the Figma mockup and not in the handoff, which is my bad to not clarify there)
- [ ] Should links to the CDC site go to a new tab? I forget if we have a norm for that on other "external" links. (Recognizing this is a case where it takes them away from ReportStream but isn't technically "external" as it is still cdc.gov)
- [ ] Under Objectives -> Support public health entities... -> the last bullet point was updated recently to: 

> Identify one or more new health care organizations, labs, or testing facilities for collaboration on their data reporting needs. 

I updated the handoff doc to reflect that update.

- [ ] I think the spacing under Roadmap is too large.
- [ ] Developer resources card - un-capitalize resources (make sentence case)

**Review by jpandersen87 — APPROVED** [review]

File: frontend-react/src/content/about/roadmap.mdx [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+import site from "../../content/site.json"
+
+<LayoutSidenav>
+<Sidenav />
+</LayoutSidenav>
+
+## Objectives for FY24
+Learn about our goals and how we are measuring success for this fiscal year (FY), which runs from October 2023 through September 2024.
+
+<hr className="rs-hr--base-lighter-alternate" />
```
Comment (etanb) [inline frontend-react/src/content/about/roadmap.mdx:23]: Small thing but for consistency: all the other About pages use `<hr className="rs-hr--alternate" />` here, which looks like:

Screenshot 2024-03-20 at 10 50 05 AM


vs. this code which looks like:

Screenshot 2024-03-20 at 10 50 33 AM

**Review by etanb — APPROVED** [review]
Small nit, but overall LGTM

File: frontend-react/src/content/about/roadmap.mdx (hunk 1 above)
Comment (penny-lischer) [reply frontend-react/src/content/about/roadmap.mdx:23]: Correct. Brought this up with design and this is how they wanted it.

**Review by audreykwr — APPROVED** [review]

**Pull request merged**